src/seedu/addressbook/ui/Formatter.java

```diff
@@ -63,4 +64,19 @@ private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
 
+
+    /**
+    * Returns a concatenated version of the printable strings of each object
+    */
+
+    public static String getPrintableString(Printable... printables){
+    StringBuilder sb = new StringBuilder();
```

Comment:
Indentation problem. Java has a StringJoiner class that can be used to join strings